Scripts/update_schema

```diff
@@ -7,9 +7,10 @@ require 'optparse'
 require 'graphql_schema'
 require 'graphql_swift_gen'
 
-endpoint = URI("https://app.shopify.com/services/ping/storefront_graphql_schema")
-body     = Net::HTTP.get(endpoint)
-schema   = GraphQLSchema.new(JSON.parse(body))
+shared_storefront_api_key = "e3e948e3697fcc37a837f7dd35719525"
```

Comment:
Just to confirm, do you intend to use the Api Key for https://app.shopify.com/services/internal/api_clients/1424832. It looks like it's the development app, not production? Is there / should there be a a production app?

Reply:
I don't think we've ever had a prod app. @davidmuzi do you have any context on this one?

Reply:
cc: @Shopify/api-patterns-team I think we would always prefer to use a production ApiClient owned by the Shopify partner, even if it's sort of a placeholder (with the correct features, betas switched on) that doesn't actually get installs. Right?

Reply:
There is no issue about sharing above key, isn't it?

Reply:
That's right. The ApiClient api key on its own can be considered public.

Reply:
This is an SDK not an app; I'm surprised it has a record at all (even for development). Chris, didn't we generate a common baseline key for SDKs to use in this case? It should have no permissions, since we can't guarantee the state of the actual apps using the SDK.

Reply:
Looks like the Unity Buy SDK is using the key from [this app](https://app.shopify.com/services/internal/api_clients/1883616). Should we do the same for iOS and Android?

Reply:
I *think* that's the right one? I'll let Chris confirm.

Reply:
Apologies for forgetting this fact. Yes, I can confirm that that is the app we should be using for all Shopify-developed Storefront API apps unless otherwise specified: https://github.com/search?q=org%3AShopify+4a6c829ec3cb12ef9004bf8ed27adf12&type=Code

Reply:
Sweet. I'll make the change.